crates/edr_evm/src/block/builder.rs

```diff
@@ -91,23 +92,25 @@ pub struct BuildBlockResult<ChainSpecT: ChainSpec> {
 }
 
 /// A builder for constructing Ethereum blocks.
-pub struct BlockBuilder<ChainSpecT: ChainSpec> {
+pub struct BlockBuilder<ChainSpecT: ChainSpec, BlockchainErrorT, StateErrorT> {
```

Comment:
Good catch! It was a leftover from when one of the trait bounds needed to know about the error type. In a second pass I was able to avoid that but never returned to the old implementation. Thanks for spotting that!

Addressed in https://github.com/NomicFoundation/edr/pull/603/commits/b608356851b96953faa3720450666e16c95da637